docs/upgrading-client.md

```diff
@@ -0,0 +1,52 @@
+<!--[metadata]>
++++
+title = "Upgrading notary client"
+description = "How to upgrade the notary client"
+keywords = ["docker, notary, notary-client"]
+[menu.main]
+parent="mn_notary"
++++
+<![end-metadata]-->
+
+# Upgrading the client
+
+### From `< v0.2` to `v0.2`
+
+Please download the v0.2 binary from [the Notary releases page](https://github.com/docker/notary/releases).
+
+##### New features:
+
+- Support for adding, editing, removing, and viewing information about [delegation roles](https://github.com/theupdateframework/tuf/blob/develop/docs/tuf-spec.txt#L252), and adding targets to delegation roles (as opposed to the targets role, which is currently the default).
+
+	This will allow repository owners to add one or more delegated users as collaborators, allowing them to use their own keys to publish.  Previous workflow, where a repository owner had share the targets key with a collaborator in order to enable them to publish.
+
+- Support for rotating the snapshot key to one managed by the Notary server.  This is necessary in order to facility delegations without forcing users to share the snapshot key with collaborators.
```

Comment:
Two small nits: `to facilitate` (instead of facility)

and `without forcing repository owners` (instead of users)